torch/onnx/symbolic_opset9.py

```diff
@@ -2371,6 +2371,19 @@ def rrelu(g, input, lower, upper, training, generator):
     return g.op('PRelu', input, p)
 
 
+def bernoulli(g, input, generator=None, out=None):
+    if generator is not None and not sym_help._is_none(generator):
+        _unimplemented("Bernoulli", "generator is not supported for bernoulli")
+
+    self = input if out is None else out
+    dtype = sym_help._try_get_scalar_type(self)
+    if dtype is None:
+        dtype = "Float"
+    p = g.op('RandomUniformLike', input, high_f=1.0, low_f=0.0)
```

Comment:
dtype is not provided for `RandomUniformLike`, thus type of `p` defaults to float. As a result, below `Less` operator will have dtype mismatch if `input` has a different type.

Reply:
Thanks, fix the RandomUniformLike result dtype.